<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
    <arg name="params_file" />
    <arg name="global_planner_params_file" />
    <arg name="output" default="screen" />
    <arg name="map_file" />

    <node name="move_base"
          pkg="move_base"
          type="move_base"
          respawn="true"
          output="$(arg output)" >
        <rosparam file="$(arg params_file)" />
        <rosparam file="$(arg global_planner_params_file)" />
    </node>

  <!-- Map server -->
  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)" output="$(arg output)"/>

</launch>
